packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -1730,109 +1730,105 @@ export class CodeModelBuilder {
     }
   }
 
-  private processSchemaFromSdkType(type: SdkType, nameHint: string): Schema {
+  private processSchema(type: SdkType, nameHint: string): Schema {
     return this.schemaCache.process(type, nameHint) || fail("Unable to process schema.");
   }
 
-  private processSchemaFromSdkTypeImpl(type: SdkType, nameHint: string): Schema {
+  private processSchemaImpl(type: SdkType, nameHint: string): Schema {
     if (isSdkBuiltInKind(type.kind)) {
-      return this.processBuiltInFromSdkType(type as SdkBuiltInType, nameHint);
+      return this.processSdkBuiltInType(type as SdkBuiltInType, nameHint);
     } else {
       switch (type.kind) {
         case "enum":
-          return this.processChoiceSchemaFromSdkType(type, type.name);
+          return this.processChoiceSchema(type, type.name);
 
         case "enumvalue":
-          return this.processConstantSchemaFromEnumValueFromSdkType(type, nameHint);
+          return this.processConstantSchemaFromEnumValue(type, nameHint);
 
         case "union":
-          return this.processUnionSchemaFromSdkType(type, type.name);
+          return this.processUnionSchema(type, type.name);
 
         case "model":
-          return this.processObjectSchemaFromSdkType(type, type.name);
+          return this.processObjectSchema(type, type.name);
 
         case "dict":
-          return this.processDictionarySchemaFromSdkType(type, nameHint);
+          return this.processDictionarySchema(type, nameHint);
 
         case "array":
-          return this.processArraySchemaFromSdkType(type, nameHint);
+          return this.processArraySchema(type, nameHint);
 
         case "duration":
-          return this.processDurationSchemaFromSdkType(
-            type,
-            nameHint,
-            getDurationFormatFromSdkType(type),
-          );
+          return this.processDurationSchema(type, nameHint, getDurationFormat(type));
 
         case "constant":
-          return this.processConstantSchemaFromSdkType(type, nameHint);
+          return this.processConstantSchema(type, nameHint);
 
         case "utcDateTime":
         case "offsetDateTime":
           if (type.encode === "unixTimestamp") {
-            return this.processUnixTimeSchemaFromSdkType(type, nameHint);
+            return this.processUnixTimeSchema(type, nameHint);
           } else {
-            return this.processDateTimeSchemaFromSdkType(type, nameHint, type.encode === "rfc7231");
+            return this.processDateTimeSchema(type, nameHint, type.encode === "rfc7231");
           }
       }
     }
     throw new Error(`Unrecognized type: '${type.kind}'.`);
   }
 
-  private processBuiltInFromSdkType(type: SdkBuiltInType, nameHint: string): Schema {
+  private processSdkBuiltInType(type: SdkBuiltInType, nameHint: string): Schema {
```

Comment:
`processBuiltInType`?

Reply:
I am fine with either. Use this name as the type to handle is `type: SdkBuiltInType`.

Updated.